content/actions/using-workflows/caching-dependencies-to-speed-up-workflows.md

```diff
@@ -360,6 +360,6 @@ jobs:
           GH_TOKEN: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
 ```
 
-Alternatively, you can use the API to programmatically delete caches on your own cadence. For more information, see "[AUTOTITLE](/rest/actions/cache#delete-github-actions-caches-for-a-repository-using-a-cache-key)."
+Alternatively, you can use the API to programmatically list or delete all caches on your own cadence. For more information, see "[AUTOTITLE](/rest/actions/cache#about-the-cache-in-github-actions)."
```

Comment:
[nitpick] Hmm...the word "programmatically" makes the sentence sound awkward for some reason. Perhaps, we can use something like automatically instead? It improves the flow of the sentence.  
```suggestion
Alternatively, you can use the API to automatically list or delete all caches on your own cadence. For more information, see "[AUTOTITLE](/rest/actions/cache#about-the-cache-in-github-actions)."
```

Reply:
Great suggestion! Thank you 🚀